components/interface/VFBTermInfo/VFBTermInfo.js

```diff
@@ -90,6 +96,28 @@ class VFBTermInfo extends React.Component {
       this.cleanButtonBar();
     }
   }
+  
+  setGraphsLinks (anyInstance) {    
+    let graphsHTML = ""
```

Comment:
As discussed, the html here should be moved to the getHTML function and changed as JSX rather than string.